festipals/src/components/SignUpForm.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export default class SignUpForm extends Component {
+
+  render() {
+    return (
+      <div>
+      <div className="Container-fluid">
```

Comment:
1. the className container-fluid shouldn't have a capital C
2. What's up with the indentation here?
3. We don't need to wrap the container-fluid inside another div, since we're only using one container here.